backtracking/graph_coloring.cpp

```diff
@@ -1,60 +1,89 @@
-#include <stdio.h>
+/**
+ * @file
+ * @brief brief one line description here
```

Comment:
Please update descriptions

Reply:
please do not resolve a conversation or comment until it is done. The suggested changes were not made and hence the conversation is not yet resolved.

Reply:
It is already fixed.

Reply:
![image](https://user-images.githubusercontent.com/7001608/87571616-5153da80-c698-11ea-951c-1ab3548226a9.png)

Reply:
Fixed.